### PR TITLE
Modified principle 9

### PR DESCRIPTION
**Pull request by fterpstra**
Added a detailed description of doing CRS content negotiation using HTTP headers

### CODE REVIEW

**Comment by fterpstra** [comment]
this pull request is in line with https://github.com/opengeospatial/conneg-by-crs RFC for Content Negotiation by CRS for IETF and the dutch API design rules GEO extension

File: README.md [hunk 1]
```diff
@@ -193,6 +193,24 @@ An example for content negotiation based on HTTP headers and with query paramete
    accept: application/json
    => should return the response using the GML encoding
 
+#### Content negotiation for coordinate reference systems (CRS)
+Request and response may be based on another coordinate reference system. This applies the HTTP-mechanism for content negotiation. The CRS of the geometry in the request (request body) is specified using the header `Content-Crs`. Below are examples for four coordinate reference systems any other CRS can be supported in the same manner.
+
+|HTTP header|Value|Explanation|
+|-|-|-|
+|`Content-Crs`|EPSG:4326|WGS84, global|
```
Comment (cportele) [inline README.md:201]: The [draft OGC spec](https://github.com/opengeospatial/conneg-by-crs/blob/master/18-072-Content-Negotiation-by-CRS.md) requires URIs a values. I.e., the value should be `http://www.opengis.net/def/crs/EPSG/0/4326`, not `EPSG:4326`.

Comment (cportele) [reply README.md:201]: It should also be added that this is work-in-progress and neither an approved standard in IETF / OGC nor widely implemented. There should not be an implication that at this time this is the preferred or only way to request / state the CRS of content.

**Comment by fterpstra** [comment]
discussed in web api guidelines 22-01-2021 because https://github.com/opengeospatial/conneg-by-crs has not progressed in two years it does not seem wise to include this in the Web API guidelines at this moment. Use of non registered headers is a bad practice and have a risk of being discarded. Can be reconsidered once it has been approved by IETF.

**Pull request closed without merging**